tests/backup/backup_license_test.go

```diff
@@ -0,0 +1,132 @@
+			destinationClusterWorkerNodes = node.GetWorkerNodes()
+			log.InfoD("Total number of worker nodes in destination cluster are %v", len(destinationClusterWorkerNodes))
+			totalNumberOfWorkerNodes = append(totalNumberOfWorkerNodes, destinationClusterWorkerNodes...)
+			log.InfoD("Total number of worker nodes in source and destination cluster are %v", len(totalNumberOfWorkerNodes))
+			log.InfoD("Switching cluster context back to source cluster")
+			err := SetSourceKubeConfig()
+			log.FailOnError(err, "Switching context to source cluster")
+		})
+		Step("Verifying the license count after adding source and destination clusters", func() {
+			log.InfoD("Verifying the license count after adding source and destination clusters")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			err = LicenseConsumedCountVerify(ctx, orgID, int64(len(totalNumberOfWorkerNodes)))
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying the license count when source cluster with %d worker nodes and destination cluster with %d worker nodes are added to backup", len(sourceClusterWorkerNodes), len(destinationClusterWorkerNodes)))
+		})
+		Step("Verify worker node on application cluster with label portworx.io/nobackup=true is not counted for licensing", func() {
+			log.InfoD("Applying label portworx.io/nobackup=true to one of the worker node on source cluster and verifying the license count")
+			err := Inst().S.AddLabelOnNode(sourceClusterWorkerNodes[0], "portworx.io/nobackup", "true")
+			log.FailOnError(err, fmt.Sprintf("Failed to apply label portworx.io/nobackup=true to worker node %v", sourceClusterWorkerNodes[0].Name))
+			sourceClusterWorkerNodeLabelApplied = true
+			sourceClusterWorkerNodeLabelRemoved = false
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			err = LicenseConsumedCountVerify(ctx, orgID, int64(len(totalNumberOfWorkerNodes)-1))
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying license count after applying label portworx.io/nobackup=true to node %v", sourceClusterWorkerNodes[0].Name))
+			log.InfoD("Switching cluster context to destination cluster")
+			SetDestinationKubeConfig()
+			log.InfoD("Applying label portworx.io/nobackup=true to one of the worker node on destination cluster and verifying the license count")
+			err = Inst().S.AddLabelOnNode(destinationClusterWorkerNodes[0], "portworx.io/nobackup", "true")
+			log.FailOnError(err, fmt.Sprintf("Failed to apply label portworx.io/nobackup=true to worker node %v", destinationClusterWorkerNodes[0].Name))
+			destinationClusterWorkerNodeLabelApplied = true
+			destinationClusterWorkerNodeLabelRemoved = false
+			log.InfoD("Switching cluster context back to source cluster")
+			err = SetSourceKubeConfig()
+			log.FailOnError(err, "Switching context to source cluster")
+			err = LicenseConsumedCountVerify(ctx, orgID, int64(len(totalNumberOfWorkerNodes)-2))
```

Comment:
Here you are applying the label on the node of destination cluster but verifying on the source cluster